app/controllers/angular_template_controller.rb

```diff
@@ -0,0 +1,5 @@
+# used to serve static angular templates from under app/views/angular_template/
```

Comment:
Mainly because nothing under app/views is plural, and I wanted the route, directory and controller name to match...